context.copytostorage/addon.py

```diff
@@ -0,0 +1,57 @@
+import xbmc
+import xbmcgui
+import xbmcaddon
+import sys
+import os
+import errno
+import shutil
+
+__addon__ = xbmcaddon.Addon()
+__addonname__ = __addon__.getAddonInfo('name')
+
+def copyFileOrFolder(src, dest):
+    print("(src, dest)",src, dest)
+    xbmc.executebuiltin("Notification(\"Started Copying ...\", \"%s\")" % sys.listitem.getLabel())
+
+    try:
+        shutil.copytree(src, dest)
+    except OSError as e:
+        if e.errno == errno.ENOTDIR:
+            shutil.copy2(src, dest)
+        else:
+            xbmc.executebuiltin("Notification(\"Directory not copied\", \"%s\")" % e)
```

Comment:
would be nice to have those strings localized, i.e., as part of translations

Reply:
`xbmcgui.Dialog().notification(...)`